C4.md

```diff
@@ -0,0 +1,144 @@
+2.  All Public Contracts SHOULD have space for extensibility and experimentation.
+3.  A patch (PR) that modifies a stable Public Contract SHOULD not break existing applications unless there is overriding consensus on the value of doing this.
+4.  A patch (PR) that introduces new features SHOULD do so using new names (a new contract).
+5.  New contracts SHOULD be marked as “draft” until they are stable and used by real users.
+6.  Old contracts SHOULD be deprecated in a systematic fashion by marking them as “deprecated” and replacing them with new contracts as needed.
+7.  When sufficient time has passed, old deprecated contracts SHOULD be removed.
+8.  Old names SHALL NOT be reused by new contracts.
+9.  A new contract marked as “draft” MUST NOT be changed to “stable” until all the following conditions are met:
+    1. Documentation has been written and is as comprehensive as that of comparable contracts.
+    2. Self-tests exercising the functionality are passing.
+    3. No changes in the contract have happened for at least one public release.
+    4. No changes in the contract have happened for at least 6 months.
+    5. No veto from the Contributor(s) of the new contract and its implementation on the change of status.
+10. A new contract marked as “draft” SHOULD be changed to “stable” when the above conditions are met.
+11. The “draft” to “stable” transition status for new contracts SHOULD be tracked using the Platform issue tracker.
+
+### 2.7. Project Administration
+
+1. The project existing Maintainers SHALL act as Administrators to manage the set of project Maintainers.
+2. The Administrators SHALL ensure their own succession over time by promoting the most effective Maintainers.
+3. A new Contributor who makes correct patches (PRs), who clearly understands the project goals. After a discussion with existing Maintainers whether we SHOULD be invite a new Contributor, the new Contributor SHOULD be invited to become a Maintainer. But only after the new Contributor has demonstrated the above for a period of time (multiple correct PRs and more than 2-3 months).
+4. Administrators MAY remove Maintainers that are long inactive (~1-2 years). Mainly due to security reasons. The Maintainer can always return back, if the person wants to become Maintainer again.
+5. Administrators SHOULD remove Maintainers who repeatedly fail to apply this process accurately.
+6. Administrators SHOULD block or ban “bad actors” who cause stress and pain to others in the project. This should be done after public discussion, with a chance for all parties to speak. A bad actor is someone who repeatedly ignores the rules and culture of the project, who is needlessly argumentative or hostile, or who is offensive, and who is unable to self-correct their behavior when asked to do so by others.
+   If the majority of the existing Maintainers agrees on the removal of the “bad actor” (after giving the “bad actor” time to self-improve), it can then be the final decision.
```

Comment:
I changed it now slightly. I do mean the currently active maintainers. I think it make sense if the majority of the current active maintainers are either neutral or agrees on the removal of another maintainer, this would then be the final agreement on the decision to proceed with removal.


What do you think?

Reply:
Yep that's entirely sensible to me 👌🏾

Reply:
Thank you for becoming a sponsor! I really appreciate that!! 💜